v201705/campaign.go

```diff
@@ -122,25 +122,25 @@ type CustomParameters struct {
 type Campaign struct {
 	Id                             int64                           `xml:"id,omitempty"`
 	Name                           string                          `xml:"name"`
-	BudgetAmount                   int64                           `xml:"budget>amount>microAmount,omitempty"`
-	Status                         string                          `xml:"status"`                  // Status: "ENABLED", "PAUSED", "REMOVED"
+	BudgetAmount                   int64                           `xml:"-"`
```

Comment:
Are you sure this won't break shopping campaigns?

Reply:
So I don't think this worked before because when we tried to create campaigns through this it didn't work and budget>amount was passed blank & adwords gave an error. 

We had to create budgets separately through budgetOperation and then you have to only pass the newly created budgetId to create the campaign. I think shopping has only 1 campaign which is created manually.

But this does need confirmation from shopping.